server/game/cards/11-TCaR/YouHadOneJob.js

```diff
@@ -0,0 +1,84 @@
+const ActionCard = require('../../actioncard.js');
+
+class YouHadOneJob extends ActionCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Shootout: Change Bullet Type',
+            target: {
+                cardCondition: {
+                    location: 'play area',
+                    condition: card => card.isWanted() && card.bounty >= 3 && card.isParticipating()
+                },
+                cardType: 'dude'
+            },
+            handler: context => {
+                this.abilityContext = context;
+                this.game.promptWithMenu(context.player, this, {
+                    activePrompt: {
+                        menuTitle: 'Change to stud or draw?',
+                        buttons: [
+                            {
+                                text: 'Change to Stud',
+                                method: 'stud'
+                            },
+                            {
+                                text: 'Change to Draw',
+                                method: 'draw'
+                            }
+                        ]
+                    },
+                    source: this
+                });
+            }
+        });
+
+        this.action({
+            title: 'Resolution: Reduce Casualties',
+            playType: 'resolution',
+            handler: context => {
+                if(!context.player.isCheatin()) {
+                    this.game.promptForYesNo(context.player, {
+                        title: 'Reduce your casualties by 2?',
+                        onYes: player => {
+                            player.modifyCasualties(-2);
+                            this.game.addMessage('{0} uses {1} to reduce their casualties by 2', player, this);
+                        }
+                    });
+                }
+
+                if(!context.player.getOpponent().isCheatin()) {
+                    this.game.promptForYesNo(context.player.getOpponent(), {
+                        title: 'Reduce your casualties by 2?',
+                        onYes: player => {
+                            player.modifyCasualties(-2);
+                            this.game.addMessage('{0} uses {1} to reduce their casualties by 2', player, this);
+                        }
+                    });
+                }
```

Comment:
Since this is for all players, can be done in a loop.
```suggestion
                this.game.getPlayers().forEach(p => {
                    if(!p.isCheatin()) {
                        this.game.promptForYesNo(p, {
                            title: 'Reduce your casualties by 2?',
                            onYes: player => {
                                player.modifyCasualties(-2);
                                this.game.addMessage('{0} uses {1} to reduce their casualties by 2', player, this);
                            }
                        });
                    }
                });
```